content/docs/02-getting-started/00-alpha.mdx

```diff
@@ -0,0 +1,212 @@
+---
+title: AI SDK 5 Alpha
+description: Get started with the latest Alpha version of the AI SDK.
+---
+
+# AI SDK 5 - Alpha
+
+<Note>
+  This is an early preview — AI SDK 5 is under active development. APIs may
+  change without notice. Pin production to v4 (ai@^4).
+</Note>
+
+## Installation
```

Comment:
need to set expectations:

- only for green-field prototypes (no migrations yet)
- not for production use
- please provide feedback
- expect large breaking changes while in alpha